[2023-04-14T22:13:55.982+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pads.launchpad scheduled__2023-01-01T00:20:00+00:00 [queued]>
[2023-04-14T22:13:56.124+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pads.launchpad scheduled__2023-01-01T00:20:00+00:00 [queued]>
[2023-04-14T22:13:56.130+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T22:13:56.134+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 2
[2023-04-14T22:13:56.137+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-14T22:13:56.299+0000] {taskinstance.py:1309} INFO - Executing <Task(BashOperator): launchpad> on 2023-01-01 00:20:00+00:00
[2023-04-14T22:13:56.312+0000] {standard_task_runner.py:55} INFO - Started process 388 to run task
[2023-04-14T22:13:56.472+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pads', 'launchpad', 'scheduled__2023-01-01T00:20:00+00:00', '--job-id', '27', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp5vtd8i7g']
[2023-04-14T22:13:56.521+0000] {standard_task_runner.py:83} INFO - Job 27: Subtask launchpad
[2023-04-14T22:13:57.246+0000] {task_command.py:389} INFO - Running <TaskInstance: pads.launchpad scheduled__2023-01-01T00:20:00+00:00 [running]> on host a0b8dfb9f67f
[2023-04-14T22:13:58.170+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Amogh Srigopal
AIRFLOW_CTX_DAG_ID=pads
AIRFLOW_CTX_TASK_ID=launchpad
AIRFLOW_CTX_EXECUTION_DATE=2023-01-01T00:20:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-01T00:20:00+00:00
[2023-04-14T22:13:58.191+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-04-14T22:13:58.207+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'python /Users/amoghsrigopal/Downloads/SpaceDataPipeline/launchpad.py']
[2023-04-14T22:13:58.450+0000] {subprocess.py:86} INFO - Output:
[2023-04-14T22:14:00.415+0000] {subprocess.py:93} INFO - python: can't open file '/Users/amoghsrigopal/Downloads/SpaceDataPipeline/launchpad.py': [Errno 2] No such file or directory
[2023-04-14T22:14:00.447+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2023-04-14T22:14:00.506+0000] {taskinstance.py:1776} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/bash.py", line 197, in execute
    f"Bash command failed. The command returned a non-zero exit code {result.exit_code}."
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2023-04-14T22:14:00.527+0000] {taskinstance.py:1332} INFO - Marking task as UP_FOR_RETRY. dag_id=pads, task_id=launchpad, execution_date=20230101T002000, start_date=20230414T221356, end_date=20230414T221400
[2023-04-14T22:14:01.495+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 27 for task launchpad (Bash command failed. The command returned a non-zero exit code 2.; 388)
[2023-04-14T22:14:01.673+0000] {local_task_job.py:212} INFO - Task exited with return code 1
[2023-04-14T22:14:01.952+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
